snap/squashfs2/directory.go

```diff
@@ -0,0 +1,98 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2018 Canonical Ltd
```

Comment:
Nitptick, wrong year; nb when introducing new files we usually insert just the current year.